src/Microsoft.DiaSymReader.Converter/Utilities/SymReaderHelpers.cs

```diff
@@ -15,7 +16,9 @@ namespace Microsoft.DiaSymReader.Tools
 {
     internal static class SymReaderHelpers
     {
-        internal static readonly Guid VisualBasicLanguageGuid = new Guid("3a12d0b8-c26c-11d0-b442-00a0244a1dd2");
+        internal static readonly Guid EncStateMachineSuspensionPoints = new("8B78CD68-2EDE-420B-980B-E15884B8AAA3");
```

Comment:
nit: Was a little confusing that this was essentially a `PortableCustomDebugInfoKinds` but not named as such.

Reply:
This is just a temporary definition - will be replaced by one from Roslyn source package once changes are made in Roslyn.